components/usage-api/usage/v1/billing.proto

```diff
@@ -22,6 +24,11 @@ service BillingService {
 
   rpc CreateStripeCustomer(CreateStripeCustomerRequest) returns (CreateStripeCustomerResponse) {};
 
+  rpc CreateStripeSubscription(CreateStripeSubscriptionRequest) returns (CreateStripeSubscriptionResponse) {};
+
+  rpc SetDefaultPaymentMethodForCustomer(SetDefaultPaymentMethodForCustomerRequest) returns (SetDefaultPaymentMethodForCustomerResponse) {};
```

Comment:
I believe this should jus be part of the CreateStripeSubscription call (an implementation detail).